libs/apnos/apnos.py

```diff
@@ -0,0 +1,140 @@
+import paramiko
+
+
+class APNOS:
+
+    def __init__(self, credentials=None):
+        self.owrt_args = "--prompt root@OpenAp -s serial --log stdout --user root --passwd openwifi"
+        if credentials is None:
+            exit()
+        self.jumphost_ip = credentials['ip']  # "192.168.200.80"
+        self.jumphost_username = credentials['username']  # "lanforge"
+        self.jumphost_password = credentials['password']  # "lanforge"
+        self.jumphost_port = credentials['port']  # 22
+        self.mode = credentials['mode']  # 1 for jumphost, 0 for direct ssh
+
+    def ssh_cli_connect(self):
+        client = paramiko.SSHClient()
+        client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        print("Connecting to jumphost: %s@%s:%s with password: %s" % (
+            self.jumphost_username, self.jumphost_ip, self.jumphost_port, self.jumphost_password))
+        client.connect(self.jumphost_ip, username=self.jumphost_username, password=self.jumphost_password,
+                       port=self.jumphost_port, timeout=10, allow_agent=False, banner_timeout=200)
+
+        return client
+
+    def iwinfo_status(self):
+        client = self.ssh_cli_connect()
+        if self.mode == 1:
+            cmd = "cd %s/lanforge/lanforge-scripts/ && ./openwrt_ctl.py %s -t %s --action cmd --value \"%s\"" % (
```

Comment:
Please use f-strings. We are using Python 3 so should use its features.

Reply:
this is done now